### DISCUSSION TITLE
Need help finding source zip URL for the latest release (2.2.800)

### DISCUSSION THREAD

**Discussion by aa1633350** [eclipse-jdt/eclipse.jdt.core#1766] [Q&A]
Hi, 
I need some help finding the source URL zip for the release 2.2.800, I have tried to look at the releases and tags section but can’t make-out the convention being followed. Please help

**Comment by merks** [comment]
Just a wild guess about what exactly you are looking:

- https://download.eclipse.org/eclipse/updates/4.30/R-4.30-202312010110/plugins/
- https://www.eclipse.org/downloads/download.php?file=/eclipse/updates/4.30/R-4.30-202312010110/plugins/org.eclipse.jdt.annotation.source_2.2.800.v20231029-1039.jar

You might state which bundle/feature or whatever you are looking for...
